nexus/celery.py

```diff
@@ -6,11 +6,15 @@
 from celery import Celery
 from django.conf import settings
 from dotenv import load_dotenv
-from raven.contrib.celery import register_signal, register_logger_signal
+from raven.contrib.celery import register_logger_signal, register_signal
 
 # Set the default Django settings module for the 'celery' program.
 load_dotenv(os.path.join(os.path.dirname(os.path.dirname(__file__)), '.env'))
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'settings.development')
+# Time period in minutes before next celery beat
+BEAT_MINUTES = {
+    'social_media_posts': 30,
```

Comment:
This key can be same as task name, like `publish_posts_task`

Also, you can rename task/service like this.